docs/api.rst

```diff
@@ -178,5 +178,29 @@ The default filter of gotcha selects all libraries loaded. This function set the
 
 .. explicit external hyperlink targets
 
+---------------------------
+Using Gotcha Version Macros
+---------------------------
+
+The source version of GOTCHA is defined by the `GOTCHA_VERSION` macro which uses the XYYYZZ format.
+Here, X signifies the major version, Y is the minor version, and Z is the patch.
+Additionally, we define `GOTCHA_VERSION_MAJOR`, `GOTCHA_VERSION_MINOR`, and `GOTCHA_VERSION_PATCH` macros for convienience.
+The codes can use the macros like
```

Comment:
is it worth to mention, that this is only available since 1.0.6. and in this case a test for the existence of the macros is needed?

```c
#if define(GOTCHA_VERSION) && GOTCHA_VERSION > 100006 // this will check of version greater than 1.0.6
#endif
```

Reply:
I dont follow why the if define is needed.

Reply:
Fixed the docs to mention that it is available since 1.0.7